### ISSUE TITLE
Metadata labels

### ISSUE THREAD

**Issue by sbrn3** [czbiohub/tabula-muris-senis#6]
Hi,

I am importing the .5had files into R using Seurat and all the metadata information e.g. cell.ontology.class is stored as numbers (0 1 2 3)  for each factor rather than their names. Is there a way that I can get the name information?

**Comment by aopisco** [comment]
@sbrn3 did you get a chance to try using the new objects (updated after revision) to our public AWS bucket: https://s3.console.aws.amazon.com/s3/buckets/czb-tabula-muris-senis/Data-objects/?region=us-west-2

**Comment by CyrilLagger** [comment]
Hi, 

@sbrn3 : I have the same issue for all the .h5ad files I tried so far (Tabula Muris and other groups). There is actually a post on that on the Seurat github [here](https://github.com/satijalab/seurat/issues/1508). But no solution proposed yet. 
For now, I just have a python script that I load in R with reticulate and takes care of collecting the missing names. 

By the way,  also note that if you use ReadH5AD() directly on the Tabula Muris Senis files, you lose some information during the conversion (notably the dimensional reduction X_pca, etc). This is discussed [here](https://github.com/satijalab/seurat/issues/1427) and happens because Seurat assumes that there is no dimensional reduction if the X matrix in the h5ad file is sparse. 
So what I had to do, is to go to python to convert the TMS matrices as dense, before converting them to Seurat objects.
